src/explore-education-statistics-admin/src/services/publicationService.ts

```diff
@@ -0,0 +1,147 @@
+import { User } from '@admin/services/PrototypeLoginService';
```

Comment:
I think that this file has accidentally been re-introduced.  On master now this file has been split over various "types.ts" files in different service packages.

If something new has been introduced in this file and that's the only reason it's being re-added, let's discuss another way to support it

Reply:
When DFE-345 is merged it will be merged into this and the problem will go away.